# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/vobcopy/notinlocal.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN vobcopy-0.5.14/Makefile vobcopy-0.5.14-new/Makefile
--- vobcopy-0.5.14/Makefile	2004-11-22 20:11:49.000000000 +0100
+++ vobcopy-0.5.14-new/Makefile	2005-05-23 09:44:08.000000000 +0200
@@ -6,7 +6,7 @@
 #PREFIX += /usr/local
 #BINDIR = ${PREFIX}/bin
 #MANDIR = ${PREFIX}/man
-PREFIX += /usr/local
+PREFIX += /usr
 BINDIR = ${PREFIX}/bin
 MANDIR = ${PREFIX}/man
 LFS    = -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE
diff -ruN vobcopy-0.5.14/configure.sh vobcopy-0.5.14-new/configure.sh
--- vobcopy-0.5.14/configure.sh	2004-11-22 20:10:37.000000000 +0100
+++ vobcopy-0.5.14-new/configure.sh	2005-05-23 09:44:51.000000000 +0200
@@ -48,7 +48,7 @@
        fi
        
        if [ "$1" != "${1%help}" ]; then
-	   echo "--prefix=PREFIX                install architecture-independent files in PREFIX [/usr/local]"
+	   echo "--prefix=PREFIX                install architecture-independent files in PREFIX [/usr]"
 	   echo "--bindir=DIR                   user executables in DIR [PREFIX/bin]"
 	   echo "--mandir=DIR                   man documentation in DIR [PREFIX/bin]"
            echo "--with-dvdread-libs=DIR        directory where dvdread lib (dvd_reader.h) is installed"
@@ -61,7 +61,7 @@
 
 if [ -z $prefix_provided ]
 then
-     prefix=/usr/local
+     prefix=/usr
 fi
 
 if [ -z $mandir_provided ]
@@ -79,7 +79,7 @@
 if [ -z $libs_dir_provided ]
 then
 
-    if [ ! -e /usr/local/include/dvdread ]; then
+    if [ ! -e /usr/include/dvdread ]; then
 	if [ ! -e /usr/include/dvdread ]; then #muss hier das then hin??
 		echo "Do you have libdvdread installed? I (the script) can't
 find it"
@@ -95,8 +95,8 @@
 	fi	
     else 
     	 echo "libdvdread found"
-	 libs_dir=/usr/local/
-#	 libs_dir=/usr/local/include	 
+	 libs_dir=/usr/
+#	 libs_dir=/usr/include	 
     fi
 else
 # Remove the following if...fi if the program complains about non-existing
@@ -134,7 +134,7 @@
 #	`sysctl -n kern.osreldate 2> /dev/null` -lt 500041 ]; then
 if [ `uname -s` = FreeBSD ]; then
   if [ `sysctl -n kern.osreldate 2> /dev/null` -lt 500041 ]; then
-	  LDFLAGS="LDFLAGS += -ldvdread -L/usr/local/lib -lgnugetopt"
+	  LDFLAGS="LDFLAGS += -ldvdread -L/usr/lib -lgnugetopt"
   else
 	  LDFLAGS="LDFLAGS += -ldvdread -L$libs_dir/lib"
   fi	
@@ -155,7 +155,7 @@
 #below here are variable definitions. They get substituted in the (CC) and 
 #stuff places.
 CC     ?= gcc
-#PREFIX += /usr/local
+#PREFIX += /usr
 #BINDIR = \${PREFIX}/bin
 #MANDIR = \${PREFIX}/man
 PREFIX += $prefix
